src/tests/benchmark.rs

```diff
@@ -204,9 +204,11 @@ unsafe fn main_0(mut argc: c_int, mut argv: *mut *mut c_char) -> c_int {
     return 0 as c_int;
 }
 #[main]
```

Comment:
Do you still need this attribute?

Reply:
yes, we don't need it for `cargo bench` support but with the attribute `cargo run` continues to work as usual. The `bench_c_vs_rust.py` script relies on the latter functionality.